pluto-connector/src/main/kotlin/club/plutomc/plutoproject/connector/api/ConnectorApiProvider.kt

```diff
@@ -2,17 +2,17 @@ package club.plutomc.plutoproject.connector.api
 
 object ConnectorApiProvider {
 
-    private var _connector: Connector? = null
-    var connector: Connector
+    private var _connectionManager: ConnectionManager? = null
+    var connectionManager: ConnectionManager
         get() {
-            return checkNotNull(_connector)
+            return checkNotNull(_connectionManager)
         }
         set(value) {
-            if (_connector != null) {
+            if (_connectionManager != null) {
                 return
             }
 
-            _connector = value
+            _connectionManager = value

```

Comment:
你为啥要这么写
你可以直接改成这个样子：
```suggestion
    var connectionManager: ConnectionManager
        get() {
            return checkNotNull(field)
        }
        set(value) {
            if (field != null) {
                return
            }
            field = value
        }

```

Reply:
忽然意识到，你这个没有初始化值，所以不是很适用我这个写法

Reply:
这样的话就必须要写成 ConnectionManager?。在 get 的时候返回的还是 ConnectionManager？，依然需要用 !!. 之类的东西把它转换成非空的样子。